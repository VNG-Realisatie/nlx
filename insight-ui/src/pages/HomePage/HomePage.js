import React from 'react'
import { Typography } from '@material-ui/core'

const HomePage = () => (
    <React.Fragment>
        <Typography variant="h4" color="primary" noWrap gutterBottom>
            Home
        </Typography>
        <Typography variant="h6" color="default" gutterBottom>
            Welcome to the NLX Insight App
        </Typography>
        <Typography variant="body1" color="default" gutterBottom>
            View logs by selecting an organization using the menu items on the
            left.
        </Typography>
        <Typography variant="body2" color="default" gutterBottom>
            You can only view organization logs if you have disclosed required
            IRMA attributes. <br />
            If you haven&apos;t, you will be asked to do so.
        </Typography>
    </React.Fragment>
)

export default HomePage
