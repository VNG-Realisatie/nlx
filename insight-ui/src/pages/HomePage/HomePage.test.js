import React from 'react'
import { shallow } from 'enzyme'
import HomePage from './HomePage'

it('renders the HomePage', () => {
    const component = shallow(<HomePage />)
    expect(component).toBeTruthy()
})

it('should match the snapshot', () => {
    const component = shallow(<HomePage />)
    expect(component).toMatchSnapshot()
})
