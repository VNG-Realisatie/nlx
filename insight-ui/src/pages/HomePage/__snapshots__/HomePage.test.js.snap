// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<Fragment>
  <WithStyles(Typography)
    color="primary"
    gutterBottom={true}
    noWrap={true}
    variant="h4"
  >
    Home
  </WithStyles(Typography)>
  <WithStyles(Typography)
    color="default"
    gutterBottom={true}
    variant="h6"
  >
    Welcome to the NLX Insight App
  </WithStyles(Typography)>
  <WithStyles(Typography)
    color="default"
    gutterBottom={true}
    variant="body1"
  >
    View logs by selecting an organization using the menu items on the left.
  </WithStyles(Typography)>
  <WithStyles(Typography)
    color="default"
    gutterBottom={true}
    variant="body2"
  >
    You can only view organization logs if you have disclosed required IRMA attributes. 
    <br />
    If you haven't, you will be asked to do so.
  </WithStyles(Typography)>
</Fragment>
`;
